### ISSUE TITLE
Any chance to upgrade this to latest swagger core?

### ISSUE THREAD

**Issue by donald-jackson** [nerdErg/swagger4jaxrs#21]
Swagger UI will be potentially be incompatible with this as they move to the API doc version 2.

**Comment by aruizca** [comment]
Yes I could. I just need the time, so I cannot give you any dates. PRs are
welcome BTW :-)

Cheers,
Angel

**Comment by donald-jackson** [comment]
@aruizca hehe yep - I actually did try to do this myself but I couldn't figure it out, perhaps beyond my Grails skills :)
